Cargo.toml

```diff
@@ -9,6 +9,10 @@ edition = "2021"
 [lib]
 crate-type = ["cdylib", "rlib"]
 
+[profile.release]
+lto = true
+opt-level = 'z'
```

Comment:
without this option, the browser wasm module inflates to 2MB

Reply:
On a separate machine now, it seems I'm getting different results.
I had thought that the current version of the optimization settings would give `741K`, but on this machine:
```
❯ du ./pkg/
 4.0K   ┌── LICENSE                   │                         █ │   0%
 4.0K   ├── README.md                 │                         █ │   0%
 4.0K   ├── package.json              │                         █ │   0%
 4.0K   │ ┌── .gitignore              │             ░░░░░░░░░░░░█ │   0%
 4.0K   │ ├── content_tag.d.ts        │             ░░░░░░░░░░░░█ │   0%
 4.0K   │ ├── content_tag_bg.wasm.d.ts│             ░░░░░░░░░░░░█ │   0%
  16K   │ ├── content_tag.js          │             ░░░░░░░░░░░░█ │   1%
 1.2M   │ ├── content_tag_bg.wasm     │             █████████████ │  48%
 1.3M   ├─┴ browser                   │             █████████████ │  50%
 4.0K   │ ┌── .gitignore              │             ░░░░░░░░░░░░█ │   0%
 4.0K   │ ├── content_tag.d.ts        │             ░░░░░░░░░░░░█ │   0%
 4.0K   │ ├── content_tag_bg.wasm.d.ts│             ░░░░░░░░░░░░█ │   0%
  16K   │ ├── content_tag.cjs         │             ░░░░░░░░░░░░█ │   1%
 1.3M   │ ├── content_tag_bg.wasm     │             █████████████ │  49%
 1.3M   ├─┴ node                      │             █████████████ │  50%
 2.6M ┌─┴ pkg                         │██████████████████████████ │ 100%

```

the browser wasm is 1.2MB...

Reply:
This ended up being about 1.35MB today:
![image](https://github.com/embroider-build/content-tag/assets/199018/2bac7d1e-8b94-4a0f-a7bc-fe2492523876)

Hopefully it can shrink in the future (or I can ditch more and more babel stuff / not use `@babel/standalone`, perhaps)

Reply:
GH says less: 
![image](https://github.com/embroider-build/content-tag/assets/199018/4274dab1-9f59-48b5-8462-d0dddc309c3d)


:shrug: could be MiB vs MB or something.


but, 487kB isn't terrible since _accuracy_ is what we're doing here. I can finally divide numbers before `<template>`, which is good.

Reply:
note that I opted out of all the different allocators -- seemed too risky, the more I read about them